docs/en/engines/table-engines/mergetree-family/annindexes.md

```diff
@@ -43,7 +43,7 @@ CREATE TABLE table
 (
   id Int64,
   vectors Array(Float32),
-  INDEX index_name vectors TYPE vector_similarity(method, distance_function[, quantization, connectivity, expansion_add, expansion_search]) [GRANULARITY N]
+  INDEX index_name vectors TYPE vector_similarity(method, distance_function[, quantization, hnsw_max_connections_per_layer, hnsw_candidate_list_size_for_construction]) [GRANULARITY N]
```

Comment:
One thing that popped into my mind and it is outside of the scope of this PR: how are we planning to handle other methods, most importantly how are we going to deal with parameters for other methods? Will the parameters interpretation depend on the value of `method`?

Reply:
Yes, that would be the plan, e.g.

```sql
INDEX index_name vectors TYPE vector_similarity('hnsw', 'L2Distance', 'f32', 16, 32] -- today's syntax

INDEX index_name vectors TYPE vector_similarity('ivf', 'L2Distance', 12, 'L2Distance'] -- possible future syntax. IVF is an alternative algorithm, having different parameters (I am making these up))
```